src/core/reducers/addons.js

```diff
@@ -229,7 +228,11 @@ export function createInternalAddon(apiAddon: ExternalAddonType): AddonType {
 
   const currentVersion = apiAddon.current_version;
 
-  if (currentVersion && currentVersion.files.length > 0) {
+  if (
+    currentVersion &&
+    currentVersion.files &&
```

Comment:
One test case was failing because of this, not sure it is realistic but it does not hurt I guess.